modules/afsocket/afsocket-source.c

```diff
@@ -843,13 +843,14 @@ _sd_open_stream(AFSocketSourceDriver *self)
 {
   GlobalConfig *cfg = log_pipe_get_config(&self->super.super.super);
   gint sock = -1;
+  gpointer fd;
   if (self->connections_kept_alive_across_reloads)
     {
       /* NOTE: this assumes that fd 0 will never be used for listening fds,
        * main.c opens fd 0 so this assumption can hold */
-      sock = GPOINTER_TO_UINT(
-               cfg_persist_config_fetch(cfg, afsocket_sd_format_listener_name(self))) -
-             1;
+      fd = cfg_persist_config_fetch(cfg, afsocket_sd_format_listener_name(self));
```

Comment:
Using fd as name and declaring it a pointer is misleading

Reply:
renamed 👍